packages/contracts/contracts/L2/predeploys/OVM_ETH.sol

```diff
@@ -31,52 +30,26 @@ contract OVM_ETH is L2StandardERC20, IWETH9 {
     {}
 
 
-    /******************************
-     * Custom WETH9 Functionality *
-     ******************************/
-    fallback() external payable {
-        deposit();
+    // Temporarily disable ETH ERC20 features.
+    // Discussion here: https://github.com/ethereum-optimism/optimism/discussions/1444
+
+    function transfer(address recipient, uint256 amount) public virtual override returns (bool) {
+        revert("OVM_ETH: ERC20 features are disabled pending further community discussion.");
```

Comment:
This should be updated to say `OVM_ETH: transfer is disabled`

Reply:
This applied for each additional revert below

Reply:
Can we just make these functions private or just delete them altogether? I don't even think we should tell developers about ETH existing at this `deaddeaddead...` address (maybe this info exists in our specs, but not in our developer docs)